app/webpack/stylesheets/openstudios/application.scss

```diff
@@ -0,0 +1,6 @@
+@import "_variables.scss";
```

Comment:
Does this mean we have an isolated style sheet for this part of the app?  if so :+1:

Reply:
Yup